xbmc/utils/LangCodeExpander.cpp

```diff
@@ -1798,3 +1878,264 @@ const std::array<ISO3166_1, 245> RegionCodes = {{
+    {"NP", "NPL", "Nepal"},
+    {"NR", "NRU", "Nauru"},
+    {"NU", "NIU", "Niue"},
+    {"NZ", "NZL", "New Zealand"},
+    {"OM", "OMN", "Oman"},
+    {"PA", "PAN", "Panama"},
+    {"PE", "PER", "Peru"},
+    {"PF", "PYF", "French Polynesia"},
+    {"PG", "PNG", "Papua New Guinea"},
+    {"PH", "PHL", "Philippines"},
+    {"PK", "PAK", "Pakistan"},
+    {"PL", "POL", "Poland"},
+    {"PM", "SPM", "Saint Pierre and Miquelon"},
+    {"PN", "PCN", "Pitcairn"},
+    {"PR", "PRI", "Puerto Rico"},
+    {"PS", "PSE", "Palestine, State of"},
+    {"PT", "PRT", "Portugal"},
+    {"PW", "PLW", "Palau"},
+    {"PY", "PRY", "Paraguay"},
+    {"QA", "QAT", "Qatar"},
+    {"RE", "REU", "Réunion"},
+    {"RO", "ROU", "Romania"},
+    {"RS", "SRB", "Serbia"},
+    {"RU", "RUS", "Russian Federation"},
+    {"RW", "RWA", "Rwanda"},
+    {"SA", "SAU", "Saudi Arabia"},
+    {"SB", "SLB", "Solomon Islands"},
+    {"SC", "SYC", "Seychelles"},
+    {"SD", "SDN", "Sudan"},
+    {"SE", "SWE", "Sweden"},
+    {"SG", "SGP", "Singapore"},
+    {"SH", "SHN", "Saint Helena, Ascension and Tristan da Cunha"},
+    {"SI", "SVN", "Slovenia"},
+    {"SJ", "SJM", "Svalbard and Jan Mayen"},
+    {"SK", "SVK", "Slovakia"},
+    {"SL", "SLE", "Sierra Leone"},
+    {"SM", "SMR", "San Marino"},
+    {"SN", "SEN", "Senegal"},
+    {"SO", "SOM", "Somalia"},
+    {"SR", "SUR", "Suriname"},
+    {"SS", "SSD", "South Sudan"},
+    {"ST", "STP", "Sao Tome and Principe"},
+    {"SV", "SLV", "El Salvador"},
+    {"SX", "SXM", "Sint Maarten (Dutch part)"},
+    {"SY", "SYR", "Syrian Arab Republic"},
+    {"SZ", "SWZ", "Eswatini"},
+    {"TC", "TCA", "Turks and Caicos Islands"},
+    {"TD", "TCD", "Chad"},
+    {"TF", "ATF", "French Southern Territories"},
+    {"TG", "TGO", "Togo"},
+    {"TH", "THA", "Thailand"},
+    {"TJ", "TJK", "Tajikistan"},
+    {"TK", "TKL", "Tokelau"},
+    {"TL", "TLS", "Timor-Leste"},
+    {"TM", "TKM", "Turkmenistan"},
+    {"TN", "TUN", "Tunisia"},
+    {"TO", "TON", "Tonga"},
+    {"TR", "TUR", "Türkiye"},
+    {"TT", "TTO", "Trinidad and Tobago"},
+    {"TV", "TUV", "Tuvalu"},
+    {"TW", "TWN", "Taiwan, Province of China"},
+    {"TZ", "TZA", "Tanzania, United Republic of"},
+    {"UA", "UKR", "Ukraine"},
+    {"UG", "UGA", "Uganda"},
+    {"UM", "UMI", "United States Minor Outlying Islands"},
+    {"US", "USA", "United States of America"},
+    {"UY", "URY", "Uruguay"},
+    {"UZ", "UZB", "Uzbekistan"},
+    {"VA", "VAT", "Holy See"},
+    {"VC", "VCT", "Saint Vincent and the Grenadines"},
+    {"VE", "VEN", "Venezuela (Bolivarian Republic of)"},
+    {"VG", "VGB", "Virgin Islands (British)"},
+    {"VI", "VIR", "Virgin Islands (U.S.)"},
+    {"VN", "VNM", "Viet Nam"},
+    {"VU", "VUT", "Vanuatu"},
+    {"WF", "WLF", "Wallis and Futuna"},
+    {"WS", "WSM", "Samoa"},
+    {"YE", "YEM", "Yemen"},
+    {"YT", "MYT", "Mayotte"},
+    {"ZA", "ZAF", "South Africa"},
+    {"ZM", "ZMB", "Zambia"},
+    {"ZW", "ZWE", "Zimbabwe"}
+}};
+// clang-format on
```

Comment:
If we want to take advantage of [`std::ranges::lower_bound`](https://en.cppreference.com/w/cpp/algorithm/ranges/lower_bound) we should ensure that the values are properly sorted:
```cpp
static_assert(std::ranges::is_sorted(RegionNames, std::ranges::less{}, &ISO3166_1A2::alpha2UC));
```